core/src/banking_stage/transaction_scheduler/transaction_state_container.rs

```diff
@@ -149,6 +153,23 @@ impl TransactionStateContainer {
             .remove(id)
             .expect("transaction must exist");
     }
+
+    /// Calculate prioritization for a transaction.
+    fn calculate_prioritization(lamports_per_signature: u64, state: &TransactionState) -> u64 {
+        let priority_details = state.transaction_priority_details();
+        let priority_fee = PrioritizationFeeDetails::new(
+            PrioritizationFeeType::ComputeUnitPrice(priority_details.priority),
+            priority_details.compute_unit_limit,
+        )
+        .get_fee();
+        let signature_fees =
+            lamports_per_signature * state.transaction_ttl().transaction.signatures().len() as u64;
+        let cost = state.transaction_cost().sum();
+
+        signature_fees
+            .saturating_add(priority_fee)
+            .saturating_div(cost)
```

Comment:
There's also a subtle difference here that currently you can game cu_price and get added priority without paying additional fees. Since we are calculating priority fees here, that boosted prioritization goes away.